<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
border:5px;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>511</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">  border-radius: 5px;
  border: 0px solid;
background-color: rgb(153, 153, 153);

</string>
    </property>
    <widget class="QWidget" name="widget_10" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">align:&quot;center&quot;</string>
      </property>
      <property name="text">
       <string>RAW</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_11" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">align:&quot;center&quot;</string>
      </property>
      <property name="text">
       <string>SNV</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_12" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>371</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">align:&quot;center&quot;</string>
      </property>
      <property name="text">
       <string>MSC</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_14" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>310</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">align:&quot;center&quot;</string>
      </property>
      <property name="text">
       <string>2nd Derivative</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_16" native="true">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QSpinBox" name="spinBox">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0);

</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Gab</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_17" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QSpinBox" name="spinBox_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Segment</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_15" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">align:&quot;center&quot;</string>
      </property>
      <property name="text">
       <string>Smoothing size</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Gab</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Segment</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_18" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">align:&quot;center&quot;</string>
      </property>
      <property name="text">
       <string>1st Derivative</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_19" native="true">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QSpinBox" name="spinBox_7">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 0, 0);

</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>0</y>
         <width>20</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Gab</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_20" native="true">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>0</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <widget class="QSpinBox" name="spinBox_8">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>20</y>
         <width>51</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>0</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Segment</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>40</y>
      <width>461</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
  background: #73AD21;
  padding: 20px;
  width: 200px;
  height: 150px;
background-color: rgb(153, 153, 153);</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">text-align:&quot;center&quot;;
font: 75 12pt &quot;MS Shell Dlg 2&quot;;
</string>
     </property>
     <property name="text">
      <string>CALCULATION STEP</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">align:&quot;center&quot;</string>
      </property>
      <property name="text">
       <string>Step 1</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">text-align: &quot;center&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>---select---</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RAW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SNV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MSC</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">align:&quot;center&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Select Your Step to Calculate</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">align:&quot;center&quot;</string>
      </property>
      <property name="text">
       <string>Step 2</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>---select---</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RAW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SNV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MSC</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_9" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">align:&quot;center&quot;</string>
      </property>
      <property name="text">
       <string>Step 3</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>---select---</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RAW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SNV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MSC</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>450</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
  background: #73AD21;
  padding: 20px;
  width: 200px;
  height: 150px;
background-color: rgb(153, 153, 153);</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">align:&quot;center&quot;</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_8" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
font: 8pt &quot;Tahoma&quot;;</string>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Type your name...</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>630</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color:#aa0000;
font-size:16px;
color:#fff;</string>
    </property>
    <property name="text">
     <string>CANCEL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>630</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color:rgb(0, 170, 0);
font-size:16px;
color:#fff;</string>
    </property>
    <property name="text">
     <string>APPLY</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
